app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
-end
+    def full_title page_title = ""
+        base_title = t  "base_title"
+        page_title.empty? ? base_title : "#{page_title} | #{base_title}"
+     end
+   end
```

Comment:
you are missing end line 
![image](https://user-images.githubusercontent.com/56816494/132303519-cc2a50dd-9509-4dbe-8173-b8fe20ae66ec.png)